clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleDiskSpec.groovy

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Please avoid adding new groovy code.  Mind rewriting this in java?

Reply:
Sure, let me change it to Java.

Reply:
done